src/Illuminate/Contracts/Validation/RequiresPreviousRule.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+namespace Illuminate\Contracts\Validation;
+
+interface RequiresPreviousRule
```

Comment:
This interface has been added so the behaviour is consistent with the current `exists` rule. See https://github.com/laravel/framework/blob/835cb93346e63a33bfb3c73180a3a0e3d0e4e60c/src/Illuminate/Validation/Validator.php#L844-L856